sdk/src/dataMapper/ModuleDataMapper.ts

```diff
@@ -17,6 +17,28 @@ export default class ModuleDataMapper extends BaseDataMapper<Module, Module_filt
         description
   }`;
 
+  async findByMultiChain(
+    chainNames: string[],
+    first?: number,
+    skip?: number,
+    where?: Module_filter,
+    orderBy?: Module_orderBy,
+    orderDirection?: OrderDirection,
+  ) {
+    const modulesResult = await this.crossChainClient.MultichainModulesQuery({
+      chainNames: chainNames,
+      first: first,
+      skip: skip,
+      where: where,
+      orderBy: orderBy,
+      orderDirection: orderDirection,
+    });
+
+    const modules = JSON.parse(JSON.stringify(modulesResult.multichainModules)) as Module[];
+
+    return modules;
```

Comment:
Instead of stringify + parsing, we can have `return modulesResult.multichainModules;` if we change the type for `Module` to:
```
export type OnChainModule = {
  ...
  chainName?: string | null;
};
```

Idem for `Attestation`, `Portal` and `Schema`